ganga/GangaDirac/BOOT.py

```diff
@@ -70,7 +70,7 @@ def startDiracProcess():
     s.close()
     #Pass the port no as an argument to the popen
     serverpath = os.path.join(os.path.dirname(inspect.getsourcefile(runClient)), 'DiracProcess.py')
-    popen_cmd = [serverpath, str(PORT)]
+    popen_cmd = ['python',serverpath, str(PORT)]
```

Comment:
Could it be undefined what version of python this will give you? Does it matter?

Reply:
This will give whatever is in the path so it is undefined. I don't think it matters - there is nothing that looks to be version specific in the script.